forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -0,0 +1,329 @@
+                    *skip_checksum,
+                );
+
+                // infallible unwrap
+                let out = chain_export(params).await.map_err(handle_rpc_err).unwrap();
+
+                println!("Export completed. Snapshot located at {}", out.display());
+            }
+            Self::Fetch { snapshot_dir } => {
+                let snapshot_dir = snapshot_dir
+                    .clone()
+                    .unwrap_or_else(|| default_snapshot_dir(&config));
+                match snapshot_fetch(&snapshot_dir, config).await {
+                    Ok(out) => println!("Snapshot successfully downloaded at {}", out.display()),
+                    Err(e) => cli_error_and_die(format!("Failed fetching the snapshot: {e}"), 1),
+                }
+            }
+            Self::Dir => {
+                let dir = default_snapshot_dir(&config);
+                println!("Default snapshot dir: {:?}", dir);
+            }
+            Self::List { snapshot_dir } => {
+                list(&config, snapshot_dir);
+            }
+            Self::Remove {
+                filename,
+                snapshot_dir,
+                force,
+            } => {
+                remove(&config, filename, snapshot_dir, *force);
+            }
+            Self::Prune {
+                snapshot_dir,
+                force,
+            } => {
+                prune(&config, snapshot_dir, *force);
+            }
+        }
+    }
+}
+
+fn list(config: &Config, snapshot_dir: &Option<PathBuf>) {
+    let snapshot_dir = snapshot_dir
+        .clone()
+        .unwrap_or_else(|| default_snapshot_dir(config));
+    println!("Snapshot dir: {:?}", snapshot_dir);
+    if let Ok(dir) = fs::read_dir(snapshot_dir) {
+        println!("\nLocal snapshots:");
+        for path in dir
+            .flatten()
+            .map(|entry| entry.path())
+            .filter(|p| p.is_file())
+        {
+            if let Some(Some(filename)) = path.file_name().map(|n| n.to_str()) {
+                if filename.ends_with(".car") {
+                    println!("{filename}");
+                }
+            }
+        }
+    }
+}
+
+fn remove(config: &Config, filename: &PathBuf, snapshot_dir: &Option<PathBuf>, yes: bool) {
+    let snapshot_dir = snapshot_dir
+        .clone()
+        .unwrap_or_else(|| default_snapshot_dir(config));
+    let mut snapshot_path = snapshot_dir;
+    snapshot_path.push(filename);
+    if snapshot_path.exists()
+        && snapshot_path.is_file()
+        && snapshot_path.extension() == Some(OsStr::new("car"))
+    {
+        println!("Deleting {:?}", snapshot_path);
+        if !yes && !prompt_confirm() {
+            println!("Aborted.");
+            return;
+        }
+
+        delete_snapshot(&snapshot_path);
+    } else {
+        println!(
+                "{:?} is not a valid snapshot file path, to list all snapshots, run forest-cli snapshot list",
+                snapshot_path);
```

Comment:
debug representation of `Path` is a bit shady, what do you think about using a regular `Display` formatting with e.g. `snapshot_path.display()`?